seeing travis is failing, but unclear how to fix this
```
FAIL: changes needed but not made due to --verify
/home/travis/gopath/src/github.com/kubernetes/kubernetes/examples/ is out of date. Please run hack/update-generated-docs.sh
!!! Error in hack/verify-generated-docs.sh:28
  '"${KUBE_ROOT}/hack/after-build/verify-generated-docs.sh" "$@"' exited with status 1
Call stack:
  1: hack/verify-generated-docs.sh:28 main(...)
Exiting with status 1
make: *** [verify] Error 1
```
happy to fix whatever is failing with a pointer as why its failing

--- a/examples/flocker/README.md
+++ b/examples/flocker/README.md
@@ -47,7 +47,7 @@ A Flocker cluster is required to use Flocker with Kubernetes. A Flocker cluster 
 - *Flocker Dataset Agent(s)*: a convergence agent that modifies the cluster state to match the desired configuration;
 - *Flocker Container Agent(s)*: a convergence agent that modifies the cluster state to match the desired configuration (unused in this configuration but still required in the cluster).
 
-Read more about the [Flocker Cluster Architecture](https://docs.clusterhq.com/en/latest/concepts/architecture.html) at the [Flocker Documentation](https://docs.clusterhq.com/).
+The Flocker cluster can be installed on the same nodes you are using for Kubernetes. For instance, you can install the Flocker Control Service on the same node as Kubernetes Master and Flocker Dataset/Container Agents on every Kubernetes Slave node.
 
 It is recommended to follow [Installing Flocker](https://docs.clusterhq.com/en/latest/install/index.html) and the instructions below to set-up the Flocker cluster to be used with Kubernetes.
 
@@ -59,7 +59,9 @@ The Flocker Control Service should be installed manually on a host, . In the fut
 
 The Flocker Agents should be manually installed on *all* Kubernetes nodes. These agents are responsible for (de)attachment and (un)mounting and are therefore services that should be run with appropriate privileges on these hosts.
 
-In order for the plugin to connect to Flocker (via REST API), several environment variables must be specified on *all* Kubernetes nodes. This may be specified in an init script for the node's Kubelet service, for example.
+In order for the plugin to connect to Flocker (via REST API), several environment variables must be specified on *all* Kubernetes nodes. This may be specified in an init script for the node's Kubelet service, for example, you could store the below environment variables in a file called `/etc/flocker/env` and place `EnvironmentFile=/etc/flocker/env` into `/etc/systemd/system/kubelet.service` or wherever the `kubelet.service` file lives.
+
+The environment variables that need to be set are:
 
 - `FLOCKER_CONTROL_SERVICE_HOST` should refer to the hostname of the Control Service
 - `FLOCKER_CONTROL_SERVICE_PORT` should refer to the port of the Control Service (the API service defaults to 4523 but this must still be specified)
@@ -67,20 +69,25 @@ In order for the plugin to connect to Flocker (via REST API), several environmen
 The following environment variables should refer to keys and certificates on the host that are specific to that host.
 
 - `FLOCKER_CONTROL_SERVICE_CA_FILE` should refer to the full path to the cluster certificate file
-- `FLOCKER_CONTROL_SERVICE_CLIENT_KEY_FILE` should refer to the full path to the key file for the API user
-- `FLOCKER_CONTROL_SERVICE_CLIENT_CERT_FILE` should refer to the full path to the certificate file for the API user
+- `FLOCKER_CONTROL_SERVICE_CLIENT_KEY_FILE` should refer to the full path to the [api key](https://docs.clusterhq.com/en/latest/config/generate-api-plugin.html) file for the API user
+- `FLOCKER_CONTROL_SERVICE_CLIENT_CERT_FILE` should refer to the full path to the [api certificate](https://docs.clusterhq.com/en/latest/config/generate-api-plugin.html) file for the API user
 
 More details regarding cluster authentication can be found at the documentation: [Flocker Cluster Security & Authentication](https://docs.clusterhq.com/en/latest/concepts/security.html) and [Configuring Cluster Authentication](https://docs.clusterhq.com/en/latest/config/configuring-authentication.html).
 
 ### Create a pod with a Flocker volume
 
-**Note**: A new dataset must first be provisioned using the Flocker tools. For example, using the [Volumes CLI](https://docs.clusterhq.com/en/latest/labs/volumes-cli.html)), create a new dataset called 'my-flocker-vol' of size 10GB:
+**Note**: A new dataset must first be provisioned using the Flocker tools or Docker CLI *(To use the Docker CLI, you need the [Flocker plugin for Docker](https://clusterhq.com/docker-plugin/) installed along with Docker 1.9+)*. For example, using the [Volumes CLI](https://docs.clusterhq.com/en/latest/labs/volumes-cli.html), create a new dataset called 'my-flocker-vol' of size 10GB:
 
 ```sh
-flocker-volumes create -m name=my-flocker-vol -s 10G
+flocker-volumes create -m name=my-flocker-vol -s 10G -n <node-uuid>
+
+# -n or --node= Is the initial primary node for dataset (any unique 
+# prefix of node uuid, see flocker-volumes list-nodes)
 ```
 
 The following *volume* spec from the [example pod](flocker-pod.yml) illustrates how to use this Flocker dataset as a volume.
+
+> Note, the [example pod](flocker-pod.yml) used here does not include a replication controller, therefore the POD will not be rescheduled upon failure. If your looking for an example that does include a replication controller and service spec you can use [this example pod including a replication controller](flocker-pod-with-rc.yml)
 
 ```yaml
   volumes:
@@ -114,15 +121,28 @@ $ ls /flocker
 0cf8789f-00da-4da0-976a-b6b1dc831159
 ```
 
+You can also see the mountpoint by inspecting the docker container on that host.
+
+```sh
+$ docker inspect -f "{{.Mounts}}" <container-id> | grep flocker
+...{ /flocker/0cf8789f-00da-4da0-976a-b6b1dc831159 /usr/share/nginx/html true}
+```
+
 Add an index.html inside this directory and use `curl` to see this HTML file served up by nginx.
 
 ```sh
-
+$ echo "<h1>Hello, World</h1>" | tee /flocker/0cf8789f-00da-4da0-976a-b6b1dc831159/index.html
 $ curl ip
 
 ```
 
+### More Info
 
+Read more about the [Flocker Cluster Architecture](https://docs.clusterhq.com/en/latest/concepts/architecture.html) and learn more about Flocker by visiting the [Flocker Documentation](https://docs.clusterhq.com/).
+
+#### Video Demo
+
+To see a demo example of using Kubernetes and Flocker, visit [Flocker's blog post on High Availability with Kubernetes and Flocker](https://clusterhq.com/2015/12/22/ha-demo-kubernetes-flocker/)
 
 <!-- BEGIN MUNGE: GENERATED_ANALYTICS -->
 [![Analytics](https://kubernetes-site.appspot.com/UA-36037335-10/GitHub/examples/flocker/README.md?pixel)]()

--- a/examples/flocker/flocker-pod-with-rc.yml
+++ b/examples/flocker/flocker-pod-with-rc.yml
@@ -1,0 +1,47 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: flocker-ghost
+  labels:
+    app: flocker-ghost
+spec:
+  ports:
+    # the port that this service should serve on
+  - port: 80
+    targetPort: 80
+  selector:
+    app: flocker-ghost
+---
+apiVersion: v1
+kind: ReplicationController
+metadata:
+  name: flocker-ghost
+  # these labels can be applied automatically
+  # from the labels in the pod template if not set
+  labels:
+    purpose: demo
+spec:
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: flocker-ghost
+    spec:
+      containers:
+      - name: flocker-ghost
+        image: ghost:0.7.1
+        env:
+        - name: GET_HOSTS_FROM
+          value: dns
+        ports:
+        - containerPort: 2368
+          hostPort: 80
+          protocol: TCP
+        volumeMounts:
+          # name must match the volume name below
+          - name: ghost-data
+            mountPath: "/var/lib/ghost"
+      volumes:
+        - name: ghost-data
+          flocker:
+            datasetName: my-flocker-vol

